tests/test_feedstock_io.py

```diff
@@ -46,7 +45,7 @@ def parameterize():
                 shutil.rmtree(tmp_dir)
 
 
-class TestFeedstockIO(unittest.TestCase):
+class TestFeedstockIO:
```

Comment:
Unit test executes the setup method before tests. If we simply remove the class inheritance here, I am not sure if that will happen or not given the code now. Can we convert the setup to a test fixture and the methods to functions?